README.md

```diff
@@ -23,11 +23,11 @@ To get started:
 ```bash
 git clone git@github.com:tinacms/tinacms.git
 cd tinacms
-npm run bootstrap
+npm install && npm run bootstrap
 
 # Start the Gatsby demo
-cd packages/demo/demo-gatsby
-npm run start
+cd packages/demo-gatsby
+npm install && npm run start
```

Comment:
Thanks a lot for the PR @adamsilverstein 😄  

This `npm install` will actually break the development environment. It will replace all the local links with the actual npm packages. The issue you hit was due to an issue with the demo site not referencing the local version numbers. I have #314 open to prevent that from happening again, and I will fix that blow up shortly.

```suggestion
npm run start
```
```

Reply:
I'll get this updated after merge. Thanks @adamsilverstein!